site/news/avatica-go-releases/index.html

```diff
@@ -0,0 +1,28 @@
+---
+layout: news
+title: Avatica Go Releases
+permalink: /news/avatica-go-releases/
+author: all
```

Comment:
Is `author` needed here?

Reply:
This was a copied from the original downloads page.